client/src/components/PostBody/PostBody.css

```diff
@@ -0,0 +1,49 @@
+.PostBody {
+  background: #fafafa;
+  width: 50rem;
+  font-family: "Open Sans", sans-serif;
+  color: #252525;
```

Comment:
Can you please use the CSS variables for color and font properties?

Reply:
Sounds good. I asked Bill to add Open Sans to the app.css. Once his PR gets merged, I can add that variable here.